.gitignore

```diff
@@ -6,6 +6,7 @@
 /pkg/
 /spec/reports/
 /tmp/
+Gemfile.lock
```

Comment:
This change is unacceptable.
Bundler recommends in its official documentation that Gemfile.lock be managed by Git, and in fact the rubocop-rails gem and rubocop-rspec gem have various problems by not managing Gemfile.lock with Git. If you want to make sure that it works with the latest version of RuboCop without any problems, I think it would be better to set up a separate workflow to check it.